### ISSUE TITLE
ModuleLoadError: [MODULE_NOT_FOUND] require failed to load 

### ISSUE THREAD

**Issue by takaHAL** [Force-Config-Control/lightning-flow-scanner-sfdx#46]
Thanks for the great plugin.
They have greatly increased the productivity of our team.
I tried to make use of it further by incorporating it into GitHub Actions, but I got a "ModuleLoadError". Do you know of any solutions?

Install CLI & plugin

```
wget https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-x64.tar.xz
  wget https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-x64.tar.xz
  mkdir ./tmp/sfdx-cli
  tar xJf sfdx-linux-x6[4](https://github.com/org/salesforce-dx/pull/11/checks#step:4:4).tar.xz -C ./tmp/sfdx-cli --strip-components 1
  echo "./tmp/sfdx-cli/bin" >> $GITHUB_PATH

 echo y | sfdx plugins:install lightning-flow-scanner
```

Error

```
$ sfdx flow:scan -p {path}

  ModuleLoadError: [MODULE_NOT_FOUND] require failed to load 
 ›   /home/runner/.local/share/sfdx/node_modules/lightning-flow-scanner/src/com
 ›   mands/flow/scan: Cannot find module '/home/runner/.local/share/sfdx/node_m
 ›   odules/lightning-flow-scanner/src/commands/flow/scan'
 ›   Require stack:
 ›   - /home/runner/work/salesforce-dx/salesforce-dx/tmp/sfdx-cli/node_modu
 ›   les/@oclif/core/lib/module-loader.js
 ›   - /home/runner/work/salesforce-dx/salesforce-dx/tmp/sfdx-cli/node_modu
 ›   les/@oclif/core/lib/help/util.js
 ›   - /home/runner/work/salesforce-dx/salesforce-dx/tmp/sfdx-cli/node_modu
 ›   les/@oclif/core/lib/help/formatter.js
 ›   - /home/runner/work/salesforce-dx/salesforce-dx/tmp/sfdx-cli/node_modu
 ›   les/@oclif/core/lib/help/command.js
 ›   - /home/runner/work/salesforce-dx/salesforce-dx/tmp/sfdx-cli/node_modu
 ›   les/@oclif/core/lib/help/index.js
 ›   - /home/runner/work/salesforce-dx/salesforce-dx/tmp/sfdx-cli/node_modu
 ›   les/@oclif/core/lib/flags.js
 ›   - /home/runner/work/salesforce-dx/salesforce-dx/tmp/sfdx-cli/node_modu
 ›   les/@oclif/core/lib/cli-ux/styled/table.js
 ›   - /home/runner/work/salesforce-dx/salesforce-dx/tmp/sfdx-cli/node_modu
 ›   les/@oclif/core/lib/cli-ux/styled/index.js
 ›   - /home/runner/work/salesforce-dx/salesforce-dx/tmp/sfdx-cli/node_modu
 ›   les/@oclif/core/lib/cli-ux/index.js
 ›   - /home/runner/work/salesforce-dx/salesforce-dx/tmp/sfdx-cli/node_modu
 ›   les/@oclif/core/lib/command.js
 ›   - /home/runner/work/salesforce-dx/salesforce-dx/tmp/sfdx-cli/node_modu
 ›   les/@oclif/core/lib/index.js
 ›   - 
 ›   /home/runner/work/salesforce-dx/salesforce-dx/tmp/sfdx-cli/dist/cli.js
 ›   - /home/runner/work/salesforce-dx/salesforce-dx/tmp/sfdx-cli/bin/run
 ›   Code: MODULE_NOT_FOUND
Error: Process completed with exit code 1.
```

**Comment by takaHAL** [comment]
> First of all, thank you for your kind words. I'd be happy to connect and see how the app can continue to help you. I can definitely think of some dependencies that might be missing locally on your machine executing github actions that haven't properly been documented(are these azure vms?). I will make some time to see if I can get something running with github actions, but if there's anything you might want to share about your specifics I'll can make myself available to that too.

I am running GitHub Actions with ubuntu-latest.

"lightning-flow-scanner" v1.1.0 was causing the above error, but after using v2.2.0, the error was resolved, thank you.

**Comment by RubenHalman** [comment]
@takaHAL my apologies. I think this was a bug we have now fixed with the latest version 2.2. Please let me know if you have any further issues!

**Comment by RubenHalman** [comment]
Glad to hear. Happy to keep in touch and hear on any feedback you might have. Thank you very much for your contribution.
